src/runtime_src/core/pcie/linux/device_linux.cpp

```diff
@@ -350,6 +350,7 @@ initialize_query_table()
   emplace_sysfs_get<query::mig_ecc_ue_ffa>              ("mig", "ecc_ue_ffa");
   emplace_sysfs_get<query::flash_bar_offset>            ("flash", "bar_off");
   emplace_sysfs_get<query::is_mfg>                      ("", "mfg");
+  emplace_sysfs_get<query::mfg_ver>                     ("", "mfg_ver");
```

Comment:
Is there a difference between mfg and mfg_ver?

Reply:
mfg is indicating if a board is running golden image. mfg_ver is the version of the golden image.